automation/src/test/scala/org/broadinstitute/dsde/firecloud/pages/AuthenticatedPage.scala

```diff
@@ -15,10 +15,20 @@ abstract class AuthenticatedPage(implicit webDriver: WebDriver) extends FireClou
     ui.clickSignOut()
   }
 
+  def filter(searchText: String): Unit = {
```

Comment:
I don't think this is general enough that it should be in `AuthenticatedPage`. It looks like you added it for a test of managing methods on workspaces so I think this should move into `WorkspaceMethodConfigPage`.